src/Appwrite/Utopia/Response/Model/Func.php

```diff
@@ -118,4 +120,22 @@ public function getType():string
     {
         return Response::MODEL_FUNCTION;
     }
+
+    /**
+     * Get Collection
```

Comment:
Docblock is wrong here.

Reply:
It would also be great to document what this filter is trying to do. (Fixing PHP default value to be {} instead of [])